pom.xml

```diff
@@ -77,6 +85,31 @@
                 <groupId>org.springframework.boot</groupId>
                 <artifactId>spring-boot-maven-plugin</artifactId>
             </plugin>
+            <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-compiler-plugin</artifactId>
+            <configuration>
+            <source>{java.version}</source>
+            <target>{java.version}</target>
+            <annotationProcessorPaths>
+                <path>
+                    <groupId>org.mapstruct</groupId>
+                    <artifactId>mapstruct-processor</artifactId>
+                    <version>${version.mapstruct}</version>
+                </path>
```

Comment:
if I'm not mistaken here we should specify the `mapstruct` dependency, not `mapstruct-processor`

however I'm not too confident how there annotation processors work :D